pubspec.yaml

```diff
@@ -7,7 +7,7 @@ homepage: https://github.com/dart-lang/dartdoc
 environment:
   sdk: '>=1.14.0 <2.0.0'
 dependencies:
-  analyzer: 0.27.3-alpha.7
+  analyzer: 0.27.4-alpha.8
```

Comment:
Yeah, I think dartdoc from the sdk will be broken, since it doesn't use the pubspec constraints but the latest analyzer source in the sdk repo. If you run dartdoc via:

```
pub global activate dartdoc
pub global run dartdoc
```

_that_ version should work (and still be pegged to the older alpha analyzer release).